README.md

```diff
@@ -110,6 +110,9 @@ teams:
         - aanm
         - borkmann
         - joestringer
+        # Optional list of team mentors who will be excluded from team review notifications
```

Comment:
There's another GitHub feature for "team review notifications" that will not be changed by this configuration. The intent here is specifically about review assignment.

```suggestion
        # Optional list of team mentors who will not be auto-assigned PRs for review
```